llvm/tools/llvm-exegesis/lib/AArch64/Target.cpp

```diff
@@ -207,18 +216,35 @@ class ExegesisAArch64Target : public ExegesisTarget {
 
     if (isPointerAuth(Opcode)) {
 #if defined(__aarch64__) && defined(__linux__)
+      // For some systems with existing PAC keys set, it is better to
+      // check the existing state of the key before setting it.
+      // For systems without PAC, this is a No-op but with PAC, it is
+      // safer to check the existing key state and then disable/enable them.
```

Comment:
Could you describe within the comment what "safer" means?

Reply:
Yes this is updated now in the comment section . Thanks